blog/app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class StaticPagesController < ApplicationController
+  def home; end
+  def help; end
```

Comment:
thêm line giữa các block, method em nhé
```suggestion
  def home; end
  
  def help; end
```